src/App.tsx

```diff
@@ -1,61 +1,38 @@
 import './App.scss';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import todosFromServer from './api/todos';
+import { TodoList } from './components/TodoList';
+import { TodosWithUsers } from './type.ts/TodosWithUser';
+import { TodoForm } from './components/TodoForm';
+import { getUserById } from './utils/userId';
+import { useState } from 'react';
+import usersFromServer from './api/users';
+
+const todos: TodosWithUsers[] = todosFromServer.map(todo => ({
+  ...todo,
+  user: getUserById(todo.userId) || null,
```

Comment:
The `getUserById` function should handle cases where it might return `null`. Consider adding a check to handle this scenario to prevent potential issues when adding todos or rendering the `TodoList`.